.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+Make sure these boxes are checked before submitting the pull request - Thank you for the contribution.
+
+- [ ] no merge conflicts
+- [ ] neither `-n` nor `--no-verify` flag was not used with `git commit`
```

Comment:
I'm not positive on this one. I'd actually tell people that this flag exists in case tests don't pass and they need help. Maybe we can take this formulation into account here?